doc/release-notes.md

```diff
@@ -23,492 +26,644 @@ shut down (which might take a few minutes for older versions), then run the
+create a wallet directory at the specified location (containing a wallet.dat
+data file, a db.log file, and database/log.?????????? files) instead of just
+creating a data file at the path and storing log files in the parent
+directory. This should make backing up wallets more straightforward than
+before because the specified wallet path can just be directly archived without
+having to look in the parent directory for transaction log files.
+
+For backwards compatibility, wallet paths that are names of existing data files
+in the `--walletdir` directory will continue to be accepted and interpreted the
+same as before.
+
+When Dash Core is not started with any `--wallet=<path>` options, the name of
+the default wallet returned by `getwalletinfo` and `listwallets` RPCs is
+now the empty string `""` instead of `"wallet.dat"`. If Dash Core is started
+with any `--wallet=<path>` options, there is no change in behavior, and the
+name of any wallet is just its `<path>` string.
+
+PrivateSend coin management improvements
 ----------------------------------------
-Version 0.14 introduced the new LLMQ-based InstantSend system which is designed to be much more scalable
-than the legacy one without sacrificing security. The new system also allows all transactions to be treated as
-InstantSend transactions. The legacy system was disabled together with the successful deployment of ChainLocks,
-but we had to keep supporting the legacy system for a while to ensure a smooth transition period. This version finally drops
-the legacy system completely.
+A new algorithm for creation of mixing denominations was implemented which
+should reduce the number of the smallest inputs created and give users more
+control on soft and hard caps. A much more accurate fee management was
+also implemented which should fix issues for users who have custom fees
+specified in wallet config or in times when network is under a stress.
+
+There is a new tab in GUI called "PrivateSend" which allows spending fully
+mixed coins only. CoinControl feature was also improved to display coins
+based on the tab it was opened at, to protect users from accidentally spending
+mixed and non-mixed coins in one transaction and to give better overview of
+spendable mixed coins.
+
+PrivateSend Random Round Mixing
+-------------------------------
+Some potential attacks on PrivateSend assume that all inputs had been mixed
+for the same number of rounds (up to 16). While this assumtion alone is not
+enough to break PrivateSend privacy, it could still provide some additional
+information for other methods like cluster analysis and help to narrow results.
+
+With Random Round Mixing, an input will be mixed to N rounds like prior. However,
+at this point, SHA256(input, salt) will be calculated and roughly a half of inputs
+will be mixed for one more round and this rule will be applied to newly created
+inputs again and so on. This results in an exponential decay where if you mix a set
```

Comment:
@thephez I feel like this could be worded better, but I'm not sure how

Reply:
I don't think it's too bad, but it isn't clear from this description what role the SHA256 calculation plays in the process.